cmd/cbuild/commands/root.go

```diff
@@ -220,15 +221,15 @@ func NewRootCmd() *cobra.Command {
 	rootCmd.Flags().BoolP("frozen-packs", "", false, "Pack list and versions from cbuild-pack.yml are fixed and raises errors if it changes")
 	rootCmd.Flags().StringP("generator", "g", "Ninja", "Select build system generator")
 	rootCmd.Flags().StringSliceP("context", "c", []string{}, "Input context names [<project-name>][.<build-type>][+<target-type>]")
-	rootCmd.Flags().StringP("load", "l", "", "Set policy for packs loading [latest | all | required]")
-	rootCmd.Flags().IntP("jobs", "j", 0, "Number of job slots for parallel execution")
+	rootCmd.Flags().StringP("load", "l", "required", "Set policy for packs loading [latest | all | required]")
+	rootCmd.Flags().IntP("jobs", "j", 8, "Number of job slots for parallel execution")
```

Comment:
The default is not `8` but `runtime.NumCPU()`
https://github.com/Open-CMSIS-Pack/cbuild/blob/v2.4.0/pkg/builder/interface.go#L107

Should this change be applied also to `SetUpCmd` in setup.go?

Reply:
setup command usage should be completely worked on as many of the options are not relevant as stated here
https://github.com/Open-CMSIS-Pack/devtools/issues/1322#issuecomment-1966244087

I would handle that in a different change request

Reply:
[Error condition to handle an invalid number of jobs](https://github.com/Open-CMSIS-Pack/cbuild/pull/258/commits/86b6ba856013794c02a471f4b4ae04e1d3f6e12d)